sdk/datafactory/Azure.ResourceManager.DataFactory/src/autorest.md

```diff
@@ -225,7 +225,7 @@ rename-mapping:
   PurviewConfiguration: FactoryPurviewConfiguration
   RunFilterParameters: RunFilterContent
   SecretBase: FactorySecretBaseDefinition
-  SecureString: FactorySecretString
+  SecureString: FactorySecureString
```

Comment:
Would it be just FactorySecretString in that case? All of the other types are prefixed with Factory.

Reply:
I see that we are using Factory in other places.  Lets track what prefix we want to use in another issue, this PR is fine to use `Factory` since that seems to be the "current prefix" choice.